tests/OBSLocal.py

```diff
@@ -331,6 +381,17 @@ def create_project(self, name, reviewer={}, maintainer={}, project_links=[]):
         return self.projects[name]
 
     def submit_package(self, package=None, project=None):
+        """Creates submit request from package to target project.
+
+        Both have to exist, otherwise looks at create_submit_request method.
+
+        :param package: package to submit TODO: what means None here?
```

Comment:
Looks like mandatory

Reply:
yes, I have committed a fix to that effect